testrunner-running/src/main/scala/edu/illinois/cs/testrunner/util/ExecutionInfo.scala

```diff
@@ -33,6 +33,7 @@ case class ExecutionInfo(classpath: String, javaAgent: Option[Path],
     def args(args: String*): List[String] =
         List("java", "-cp", classpath) ++
         javaAgent.map(p => List("-javaagent:" ++ p.toAbsolutePath.toString)).getOrElse(List.empty) ++
+        List("-Xms4096M", "-Xmx4096M") ++
```

Comment:
For now, let's not force the maximum memory, but we should introduce it as an option later.